Modix.Services/Notes/NoteCreatorService.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Modix.Services.Notes
+{
+    using System;
+    using System.Threading.Tasks;
+    using Data;
+    using Data.Models.Moderation;
+    using Discord;
+    using Serilog;
+
+    public class NoteCreatorService : INoteCreatorService
+    {
+        private readonly ModixContext _modixContext;
+
+        public NoteCreatorService(ModixContext modixContext)
+        {
+            _modixContext = modixContext;
+        }
+
+        public async Task<bool> Create(IGuildUser user, string message, string recordedBy)
+        {
+            try
+            {
+                var note = new NoteEntity
+                {
+                    UserName = user.Username,
+                    UserId = user.Id,
+                    Message = message,
+                    RecordedTime = DateTime.UtcNow,
+                    RecordedBy = recordedBy
+                };
+
+                await _modixContext.Notes.AddAsync(note);
+                var rowsAdded = await _modixContext.SaveChangesAsync();
+
+                return rowsAdded > 0;
+            }
+            catch (Exception e)
+            {
+                Log.Error(e, $"AddNoteService Create has exceptioned with the following user: {user.Username} and message: {message}");
```

Comment:
Minor nitpick, but with Serilog it's preferred to rely on [message templates](https://github.com/serilog/serilog/wiki/Writing-Log-Events) for formatting the log message rather than using concatenation, interpolation, or string.Format yourself.

We already go against this recommendation in several other places throughout the code base so it's not a big deal (and definitely not a blocker).

Reply:
I didn't know that, from now on i'll start using message templates because those are pretty nifty.